src/components/testCaseLanding/qdm/TestCaseList.tsx

```diff
@@ -125,6 +132,21 @@ const TestCaseList = (props: TestCaseListProps) => {
   const [importDialogState, setImportDialogState] = useState<any>({
     open: false,
   });
+  const cqlParsingService = useRef(useCqlParsingService());
+
+  const [callstackMap, setCallstackMap] = useState<CqlDefinitionCallstack>();
+
+  useEffect(() => {
+    cqlParsingService.current
+      .getDefinitionCallstacks(measure.cql)
+      .then((callstack: CqlDefinitionCallstack) => {
+        setCallstackMap(callstack);
+        return callstack;
+      })
+      .catch((error) => {
+        console.error(error);
```

Comment:
Since there a service call, Do you think we should display an error message to the user ? A generic error message atleast ?